learning_resources/serializers.py

```diff
@@ -26,6 +27,7 @@ class Meta:
         fields = ["id", "name"]
 
 
+@extend_schema_field({"type": "array", "items": {"type": "string"}})
```

Comment:
For SerializerMethodFields, a normal type annotation on the method is enough for drf spectacular. But not for `serializers.Field` :(. See https://github.com/tfranzel/drf-spectacular/issues/886